.dependabot/config.yml

```diff
@@ -11,3 +11,13 @@ update_configs:
     - form-builder-developers
     - tomas-stefano
     - brenetic
+  - package_manager: "docker"
+    directory: "/"
+    update_schedule: "daily"
+    default_reviewers:
+    - asmega
+    - sequencemedialimited
+    - mattempty
+    - form-builder-developers
+    - tomas-stefano
+    - brenetic
```

Comment:
just out of interest does dependabot support yaml anchoring?

`default_reviewers: &reviewers` && `default_reviewers: *reviewers`

Reply:
probably. i don' think it makes a huge difference though. feel free to make the PR :)